tutorial.md

```diff
@@ -104,6 +104,9 @@ go version go1.8.5 linux/amd64
 
 ```
 go get github.com/kubernetes-incubator/cri-tools/cmd/crictl
+cd ./src/github.com/kubernetes-incubator/cri-tools/cmd/crictl
+make 
+make install
```

Comment:
I don't think we need these, because `crictl` [claims][1] to support `go get`.

[1]: https://github.com/kubernetes-incubator/cri-tools/blob/a6dd109ef238be53ab9a7a5c4642f8a1d4ba1bd6/docs/crictl.md#install

Reply:
I thought that we didn't need these either, but when I walked through the tutorial on  a fresh f27 machine, I couldn't run crictl until I'd done these three steps.